modules/cnf-node-tuning-operator-creating-pod-with-guaranteed-qos-class.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-node-tuning-operator-creating-pod-with-guaranteed-qos-class_{context}"]
+= Creating a pod with a QoS class of `Guaranteed`
```

Comment:
Doc Guidelines: [Do not use backticks or other markup in assembly or module headings.](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings)

```suggestion
= Creating a pod with the Guranteed QoS class of Guaranteed
```
Or 
```suggestion
= Creating a pod with a Guaranteed QoS class
```